packages/mermaid/src/docs/syntax/sequenceDiagram.md

```diff
@@ -377,6 +377,30 @@ sequenceDiagram
     API-->BillingService: Start billing process
 ```
 
+## Custom Grouping (v<MERMAID_RELEASE_VERSION>+)
+
+It is possible to create a group of actions similar to those described above, but with a custom label.
+
+This is done by the notation
+
+```
+group:label [description]
+... statements ...
+end
+```
+
+See the example below:
+
+```
+sequenceDiagram
+    Alice-->Bob: I have a secret
+    Bob-->Alice: What is it?
+    group:Whisper So nobody hears
```

Comment:
```suggestion
    group:Whisper [So nobody hears]
```

Let's wrap the description in `[]`, so that we can have labels with spaces. 
`group:"Whisper" ["So nobody hears"]` Quoting support can also be added to support unicode.